### ISSUE TITLE
Getting error when adding `d3-ng2-service` with webpack for angular2

### ISSUE THREAD

**Issue by vcernomschi** [tomwanzek/d3-ng2-service#12]
Getting error when importing `d3-ng2-service` with webpack for angular2.

Steps to reproduce.
1. Create project based on `https://angular.io/docs/ts/latest/guide/webpack.html`.
2. Add `d3-ng2-service` based on `https://www.npmjs.com/package/d3-ng2-service`.
3. Add d3 chart sample from `https://github.com/tomwanzek/d3-ng2-demo` to check if it works.
Expected result: Chart is displayed without errors;
Actual result: Chart is not displayed. The following error is displayed in `d3-ng2-service/index.js` :
```
index.js: Uncaught SyntaxError: Unexpected token export
```
Just wonder if there are need any additional settings to webpack which I need to add to get it working. It looks like currently it doesn't support es6 keyword. But we have the same imports in polyfills.ts as in `d3-ng2-demo`.

Please refer to repo: https://github.com/vcernomschi/ng2-d3-webpack

If you comment out the below statements: 
1 - importing `d3-ng2-service` and declaration`BrushZoom2Component` in root module `src/app/module/app.module.ts`
2 - exporting `export * from './d3-demos/brush-zoom-2/brush-zoom-2.component';` in`src/app/components/reports/index.js`
you will get it working without `d3-ng2-service`;

Thanks in advance.

**Comment by tomwanzek** [comment]
**angular-cli** @latest (beta.15) [package.json](https://github.com/angular/angular-cli/blob/master/package.json) uses webpack 2 ("webpack": "2.1.0-beta.22"). Webpack 2 has native ES 6 import support. See the [webpack  roadmap](https://webpack.github.io/docs/roadmap.html) and their pre-releases.

The package.json for the webpack intro you are starting from is based on webpack ^1.13.0, so there would be additional ES 6 support required in the config/loaders.

Other than opting for **angular-cli** directly, if you want to roll it yourself, webpack 2 might be a an alternative starting point. I know it's beta...

**Comment by vcernomschi** [comment]
Thanks @tomwanzek. Now is working fine.

**Comment by tomwanzek** [comment]
Which road did you take? Sorry, if I did not have a ton of time for this right now. I have started working on beefing up the D3 module definitions that feed DefinitelyTyped / @types with JSDoc comments. Was one of my remaining todo's on the major re-write. A bit tedious, but ultimately worth it for the long run...

**Comment by vcernomschi** [comment]
Cool :) I updated my [package.json](https://github.com/vcernomschi/ng2-d3-webpack/commit/55ca53fd676016378a0d76c7a293f36b5a5d2c43) to have the same webpack versions as in `angular-cli`.
There are also some changes in [extract-text-webpack-plugin](https://github.com/webpack/extract-text-webpack-plugin/commit/91f6009). So I did some minor changes in  my `webpack.common.js`, because ExtractTextPlugin in beta uses one parameter.

Now I am going to create some line and bar charts from https://bl.ocks.org/mbostock

Thanks for your assistance @tomwanzek!
